src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.css

```diff
@@ -1,6 +1,11 @@
 .THIS.slds-tabs_card {
     background: white;
 }
+
+/*remove focus button formatting*/
+.THIS .slds-button:focus {
```

Comment:
Can this use a class instead? We're not using any other buttons currently, but would rather this was a more specific selector just in case we do.

Reply:
Good thought! I like that way better. 

**NB! I do want to solicit a bit of broader feedback on this topic though - this css is basically in here to make the base button look/behave more like a link, where the principal difference that I noticed is the focus/active behavior showing the shadow/border around the un-styled button. I'm curious if the team thinks that it's actually just a bad idea to try and make a button act like a link, and if we should use one of the LDS button variants without additional theming. 

Despite appearing like a link in the mockup, this is in fact a button for two reasons - one, the LDS style guide suggests using links only when the user is directed to a new url, which is not the case here. The second is that the way all the previous user interaction is being handled in the controller is by identifying the source of the user action but the aura:id attribute, which isn't available on links.

I think the intent of showing this more like a link visually in the mock-up was to make this feature more subtle than a button, so that users who want a straightforward user experience aren't distracted into clicking it and getting confused by the advanced options. However, for obvious reasons "slds-not-very-obvious-button" is not one of the brand variants. So, in summary: 

- Maybe we shouldn't style this button at all and just use the SLDS base (or neutral?) button variants, so we aren't fighting the design system to make it look like a link? 

- If we want to make it look like a link, @randi274's suggestion is definitely the way to go.

- We shouldn't actually implement it with an <a> link however

Thoughts?

Reply:
(Going with using css class because everyone likes it)